features/login-testcase.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Login view
+
+    Scenario: User opens pah-fm login view
+        When User opens pah-fm website
+        Then User sees login view with 2 text fields: "Username" and "Password" and 1 button "Login"
```

Comment:
If there are number of elements, you could use table inside the step. Not needed here though. 

```
Scenario: some scenario
  Given a set of specific users
     | name      | department  |
     | Barry     | Beer Cans   |
     | Pudey     | Silly Walks |
     | Two-Lumps | Silly Walks |

 When we count the number of people in each department
 Then we will find two people in "Silly Walks"
  But we will find one person in "Beer Cans"
```